webpack/assets/javascripts/react_app/components/HostDetails/Audits/index.js

```diff
@@ -1,143 +1,101 @@
 import PropTypes from 'prop-types';
-import React, { useState } from 'react';
-import Skeleton from 'react-loading-skeleton';
-import { ArrowIcon } from '@patternfly/react-icons';
-import ElipsisWithTooltip from 'react-ellipsis-with-tooltip';
+import React, { useState, useEffect } from 'react';
 import {
-  Card,
+  Bullseye,
   DataList,
+  Card,
+  CardActions,
+  CardHeader,
+  CardTitle,
+  CardBody,
   DataListItem,
   DataListItemRow,
   DataListItemCells,
+  DataListWrapModifier,
   DataListCell,
-  CardBody,
-  CardTitle,
-  Accordion,
-  AccordionItem,
-  AccordionContent,
-  AccordionToggle,
+  Title,
 } from '@patternfly/react-core';
+import { Link } from 'react-router-dom';
+import URI from 'urijs';
 
-import { foremanUrl } from '../../../../foreman_tools';
+import { foremanUrl } from '../../../common/helpers';
 import { translate as __ } from '../../../common/I18n';
 import { useAPI } from '../../../common/hooks/API/APIHooks';
+import RelativeDateTime from '../../common/dates/RelativeDateTime';
+import SkeletonLoader from '../../common/SkeletonLoader';
+import { STATUS } from '../../../constants';
+
+const NUMBER_OF_RECORDS = 3;
+const BASE_URL = '/audits';
 
 const AuditCard = ({ hostName }) => {
```

Comment:
Instead of waiting for the hostname to come back from the results of the first api call, can we use it from the route directly right away? Or pass it directly instead of the results (as it is available on the initial page load already)? That would make a lot of the defensive `if (hostName)`s not needed, and we won't even need to use the state for storing the urls - they would just be calculated directly from the hostname.